### PR TITLE
[BASE] Improve Spanish translation

### PR DESCRIPTION
**Pull request by jeffbox12**
## Purpose

General translation in Spanish.

## Changes

Missing translations and spell corrections.

- Missing translations from applications: `mspaint`, `notepad` & `rapps`

- Missing translations and improvements on `osk`

- Missing translations on cmd help descriptions: `assoc`, `CTTY`, `replace`, `prompt `& `set`

- Missing translation on system: `subst`, `userinit` & `winlogon`

### CODE REVIEW

File: base/applications/osk/lang/es-ES.rc [hunk 1]
```diff
@@ -239,69 +240,69 @@ BEGIN
...
     END
 END
 
 IDD_WARNINGDIALOG_OSK DIALOGEX 0, 0, 250, 97
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUP | WS_CAPTION
-CAPTION "On-Screen Keyboard"
+CAPTION "Teclado en Pantalla"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
     DEFPUSHBUTTON "OK", IDOK, 193, 76, 50, 14
```
Comment (jeffbox12) [inline base/applications/osk/lang/es-ES.rc:296]: > El mensaje que aparece al abrir Teclado en Pantalla, el botón OK parece no tener un string para traducción.
> imagen

Resulta que si que tenia string para traducir. Fallo mio.

File: base/system/userinit/lang/es-ES.rc [hunk 2]
```diff
@@ -23,24 +28,24 @@ CAPTION "ReactOS LiveCD"
...
+    PUSHBUTTON "&Cancelar", IDCANCEL, 260, 172, 50, 14
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SHELL_FAIL "Userinit no pudo iniciar el shell con éxito!"
-    IDS_INSTALLER_FAIL "Userinit failed to start the installer!"
-    IDS_CANCEL_CONFIRM "Are you sure you want to quit the ReactOS LiveCD?\nIf you choose to do so, your computer might restart."
+    IDS_SHELL_FAIL "Userinit no pudo iniciar el shell con éxito."
+    IDS_INSTALLER_FAIL "Userinit no pudo iniciar el instalador."
```
Comment (binarymaster) [inline base/system/userinit/lang/es-ES.rc:44]: These sentences end with an exclamation sign in other translations:
- https://git.reactos.org/?p=reactos.git&a=search&h=master&st=grep&s=IDS_SHELL_FAIL
- https://git.reactos.org/?p=reactos.git&a=search&h=master&st=grep&s=IDS_INSTALLER_FAIL

**Comment by julenuri** [comment]
Ready to merge

**Review by binarymaster — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Catalin Gabriel Draghita"
git config --global user.email catalin.draghita@hotmail.es
```

Or just for the current repository:
```
git config user.name "Catalin Gabriel Draghita"
git config user.email catalin.draghita@hotmail.es
```

Also please update your name in [GitHub profile settings](https://github.com/settings/profile).

Regards,

**Pull request merged**